arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender-tianma.dts

```diff
@@ -0,0 +1,10 @@
+/dts-v1/;
+#include "sdm660-xiaomi-lavender-common.dtsi"
+
+&nt36xxx_ts {
+	status = "okay";
+};
+
+&panel {
+	compatible = "tianma,nt36672a-xiaomi-lavender";
```

Comment:
for now it is `tianma,nt36672a-xiaomi-lavender-simple` still as is on target branch

Reply:
What about full driver?

Reply:
If it worth it, yes.